### PR TITLE
Display a grid of company logos in the Companies scene.

### PR DESCRIPTION
**Pull request by Lateworm**

### CODE REVIEW

File: js/scenes/Companies/CompaniesContainer.js [hunk 1]
```diff
@@ -4,18 +4,35 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
```
Comment (siddharthparmar7) [inline js/scenes/Companies/CompaniesContainer.js:2]: Please remove comment from line 1

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Thank you, Travis! good work. Can make a requested change so I can approve this pull request.

**Comment by Lateworm** [comment]
Hi Sid. The comment has been removed. Thank You!

**Review by siddharthparmar7 — APPROVED** [review]
Thank you! Travis!

**Pull request merged**